package.json

```diff
@@ -47,7 +50,7 @@
     "shell-exec": "^1.0.2",
     "ts-jest": "^23.10.4",
     "typescript": "^3.1.3",
-    "web3": "^1.0.0-beta.36",
+    "web3": "1.0.0-beta.36",
```

Comment:
is that not include in graph-cli ?

Reply:
Don't know. But anyway we also use web3 for testing and migration. Since its still unstable we need to fix the version.